code/core/package.json

```diff
@@ -254,7 +270,8 @@
     "express": "^4.19.2",
     "process": "^0.11.10",
     "recast": "^0.23.5",
-    "util": "^0.12.4",
+    "semver": "^7.6.2",
+    "tempy": "^3.1.0",
```

Comment:
I was not able to prebundle those.

Reply:
I removed the need for `tempy` earlier, also because I couldn't prebundle it.

I made my own version here:
https://github.com/storybookjs/storybook/blob/c5290faee38bd40bfd759aafe7be68e7e1dd8982/code/core/src/common/utils/cli.ts#L46-L59

https://github.com/storybookjs/storybook/blob/c5290faee38bd40bfd759aafe7be68e7e1dd8982/code/core/src/common/utils/cli.ts#L15-L19